servicemigration/context.py

```diff
@@ -11,20 +19,32 @@ class ServiceContext():
     @versioned
     def __init__(self, filename=None):
         """
-        Initializes the ServiceContext for the given filename, or the file
-        defined by sys.argv[1] if filename is None.
+        Initializes the ServiceContext. Input precedence is filename,
+        MIGRATE_INPUTFILE, servicemigration endpoint.
```

Comment:
This is the only doc on how to use this, right?  In which case, I think it's needs a little more clarification. Like `MIGRATE_INPUTFILE` is the name of an environment variable that contains the name of the json file of service definitions to load, and `CONTROLPLANE_TENANT_ID` is the name of an environment variable that contains the ID of the tenant application which will be retrieved from control center (or something to that effect)